Makefile

```
@@ -0,0 +1,56 @@
+.PHONY: build run clean test help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)